Description: Fix 'permit to' typo, replace with 'permit <verb>ing'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://trac.osgeo.org/grass/ticket/2704

--- a/raster/r.relief/r.relief.html
+++ b/raster/r.relief/r.relief.html
@@ -85,7 +85,7 @@ r.relief input=srtm output=srtm_shaded s
 
 <h3>Exporting shaded relief maps to GeoTIFF</h3>
 
-The data range of shaded relief maps usually does not permit to export the
+The data range of shaded relief maps usually does not permit exporting the
 map to GeoTIFF format along with its associated color table due to limitations
 in the GeoTIFF format.
 <p>
